plugins/accessibility/index.js

```diff
@@ -175,6 +175,15 @@ function runChromeDevTools(config) {
   var elementPromises = [],
       elementStringLength = 200;
 
+  function trimText(text) {
+    if (text.length > elementStringLength) {
+      return text.substring(0, elementStringLength / 2) + ' ... '
```

Comment:
Yeah (there's also some tabs at the beginning, so it's not the whole output string length anyway...).